### DISCUSSION TITLE
Event Throttling support

### DISCUSSION THREAD

**Discussion by subhajitm** [microsoft/cpp_client_telemetry#1149] [Q&A] [answered]
does 1DS support throttling noisy events?

**Comment by lalitb** [comment, marked as answer]
No, 1DS doesn't have inbuilt intelligence to throttle noisy events. Application can define filtering/sampling rule to throttle any such events.  More information - https://github.com/microsoft/cpp_client_telemetry/blob/main/lib/include/public/IEventFilter.hpp
